### PR TITLE
Added paragraph on how to edit files in GitHub PRs

### PR DESCRIPTION
**Pull request by Ryuno-Ki**
> Issue number that this PR fixes (if any). For example: 'Fixes #987654321'

https://github.com/mdn/content/discussions/6935

> What was wrong/why is this fix needed? (quick summary only)

We received repeated questions on how to change files using GitHub's web UI after a PR got created.
In order to not repeat ourselves, I propose to add a paragraph to the README.

> Anything else that could help us review it

Added @Rumyra as per https://github.com/mdn/content/discussions/6935#discussioncomment-1028335

### CODE REVIEW

**Review by hamishwillee — APPROVED** [review]
Looks good. I've improved the wording to my taste. How do I do this as a "suggestion" rather than a change? 
Anyway, not merged but approved, so you have opportunity to modify if you like.

**Comment by sideshowbarker** [comment]
The https://github.com/mdn/content/pull/7563/checks?check_run_id=3235774417#step:4:5 lint error “README.md:156:81 MD013/line-length Line length [Expected: 80; Actual: 200]” seems like a misfeature in the linter, right? I thought we had agreement that we didn’t want to try to enforce any line lengths for our Markdown sources?

Anyway, due to that, I guess merging this will require @Rumyra or @mdn/core-dev or somebody else with admin perms

**Comment by wbamberg** [comment]
> The https://github.com/mdn/content/pull/7563/checks?check_run_id=3235774417#step:4:5 lint error “README.md:156:81 MD013/line-length Line length [Expected: 80; Actual: 200]” seems like a misfeature in the linter, right? I thought we had agreement that we didn’t want to try to enforce any line lengths for our Markdown sources?

The linter doesn't do this for the docs content but does for MD meta-content like the README (and for the same reason it should not). See also https://github.com/mdn/content/issues/7566 which I need to get around to reviewing...

**Comment by hamishwillee** [comment]
Or one of us can edit the lines to the right length. But I'm working elsewhere today.

File: README.md [hunk 1]
```diff
@@ -152,6 +152,11 @@ with us first!
 You can [chat with us](https://chat.mozilla.org/#/room/#mdn:mozilla.org) or
 [file an issue](https://github.com/mdn/content/issues).
 
+You may be asked to edit files in your pull request.
```
Comment (teoli2003) [inline README.md:155]:
```suggestion
You may be asked to further edit files in your pull request.
```

File: README.md [hunk 2]
```diff
@@ -152,6 +152,11 @@ with us first!
...
+To open a file for editing, select the *Files changed* tab on the PR, scroll down to the section for the file you want to edit, and then select the "three dots" icon (at the top right of the section).
+Choose **Edit file** from the popup menu to start editing the file.
+After editing your changes will result in a new commit.
```
Comment (teoli2003) [inline README.md:158]:
```suggestion
After editing, your changes will result in a new commit.
```

**Review by teoli2003 — DISMISSED** [review]
Two nits and we are good to go.

**Comment by AnilSeervi** [comment]
Hello @Ryuno-Ki , I've patched up the lint error and the changes requested into a commit and unfortunately I couldn't commit to the same PR.
Is it okay if your commit gets merged from the new PR I've created #7563 ?

**Comment by Ryuno-Ki** [comment]
Wow :open_mouth: 

I create a file, go to bed, next day morning and in the evening I learn that I broke the Markdown linter :scream_cat: 

@AnilSeervi I'd like to reformat (and commit the suggestion) here. If that doesn't work out, you've got my blessing for your PR.

**Review by hamishwillee — APPROVED** [review]
I'm approving again and merging. The changes suggested in review were all addressed so this is good to go. Thanks all.

**Pull request merged**